router/config.go

```diff
@@ -80,3 +82,13 @@ type RealmConfig struct {
 	// embedding nexus.  A value of nil enables the default filtering.
 	PublishFilterFactory FilterFactory
 }
+
+// TopicEventHistoryConfig stores settings for event history for a topic
+type TopicEventHistoryConfig struct {
+	// Topic URI
```

Comment:
This comment is not providing any useful information, it's quite clear from name and type already.

Reply:
yeah. Removed